scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/goals/CoexistenceGoal.java

```diff
@@ -266,11 +277,19 @@ else if (this.initiallyEvaluatedTemporalContext == null) {
       if (!alreadyOneActivityAssociated) {
         //create conflict if no matching target activity found
         if (existingActs.isEmpty()) {
+          var temporalContext = this.temporalContext.evaluate(simulationResults, evaluationEnvironment);
+          final var activityCreationTemplateBuilt = activityCreationTemplate.build();
+          final var evaluationEnvironement = createEvaluationEnvironmentFromAnchor(evaluationEnvironment, window);
+          final var newTemporalContext = boundTemporalContextWithSchedulerModel(
```

Comment:
Why is CoexistenceGoal the only goal type that received the additional bounding when making a `MissingActivityTemplateConflict`?

Reply:
Because only in coexistence goal there is such a precise notion of where the activity must be located. Cardinality and Recurrence are vague over each of their intervals.